src/GovUk.Education.ExploreEducationStatistics.Publisher.Tests/Services/ReleaseServiceTests.cs

```diff
@@ -875,6 +914,12 @@ await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId)
                 Assert.Equal("<p>Release 2 summary 1 order 2</p>",
                     (summarySectionContent[2] as HtmlBlockViewModel)?.Body);
 
+                var relatedDashboardsSection = result.RelatedDashboardsSection;
+                Assert.NotNull(relatedDashboardsSection);
+                Assert.Equal(Release2RelatedDashboardsSection.Id, relatedDashboardsSection.Id);
```

Comment:
I think there's a warning here you can supress in `relatedDashboardsSection.Id` since we know `relatedDashboardsSection` is not null at this point.

Same problem also on line 1165.